Examples/Examples/FloorFilterExampleView.swift

```diff
@@ -18,7 +18,7 @@ import ArcGIS
 
 struct FloorFilterExampleView: View {
     /// Determines the arrangement of the inner `FloorFilter` UI components.
-    private var floorFilterAlignment: Alignment { .bottomLeading }
+    let floorFilterAlignment = Alignment.bottomLeading
```

Comment:
No reason for this to be computed. It never changes.

Reply:
Should it still be private?

Reply:
> It never changes.

That is precisely why it should be computed: so no memory is allocated for it each time the example view is instantiated. Either that or it should be static. Same with similar changes in this PR.

Reply:
Ah okay, reverted that commit. Thanks for catching!